source/api/extended.cpp

```diff
@@ -914,6 +915,45 @@ Extended::initiateTransfer(const Models::Address&               inputAddress,
   return bundle.getTransactions();
 }
 
+bool
+Extended::isPromotable(const Types::Trytes& tail) const {
+  if (!Types::isValidHash(tail)) {
+    return false;
+  }
+
+  try {
+    return checkConsistency({ tail }).getState();
+  } catch (IOTA::Errors::BadRequest) {
+    return false;
+  }
+  return false;
```

Comment:
most probably return true instead here no? Otherwise this function always returns false.

Reply:
my bad, did not read carefully enough.